### DISCUSSION TITLE
Please-support-the-application-in-Arabic ios

### DISCUSSION THREAD

**Discussion by alialhammad1992** [community/community#16262] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.64.0 (66902451)
Device: iPhone 13 Pro Max (iOS 15.5)
</details>

**Comment by alcere** [comment]
Hi @alialhammad1992 thanks for your feedback! I will share your suggestions with the mobile team.

**Comment by nasrqwertuyy** [comment]
في الأربعاء، 10 أوت 2022 3:02 م Alfredo ***@***.***> كتب:
